src/backend/tests/test_projects_routes.py

```diff
@@ -367,6 +367,33 @@ async def test_project_by_id(client, project):
     assert data["tasks"] == project.tasks
 
 
+async def test_get_odk_entities_mapping_statuses(client, project):
+    """Test get the ODK entities mapping statuses."""
+    # mock dependency on odk central library
+    entities_data_mock = [
+        {
+            "id": "455d0982-f4d9-4a08-ab94-dba22aa951b2",
+            "task_id": 1,
+            "osm_id": 302546443,
+            "status": 0,
+            "updatedAt": None,
+        }
+    ]
+
+    with patch(
```

Comment:
Nice idea with using unittest.patch, but I think this test isn't actually testing anything as a result!

If you patch the response to be what you want it to be, then testing the response equals what you set it to doesn't help much 😄 

I would recommend writing this test to incorporate both posting an entity status and then reading a single entity, and then possibly reading `/entities/statuses` after to check it returns correctly as a list of entities too.

Reply:
@spwoodcock, I have addressed this in the last commit and added additional tests for updating and retrieving the status of a single entity.

The PR description and title have been updated accordingly, please have look. Thanks!🙏